automatic/tightvnc/update.ps1

```diff
@@ -0,0 +1,30 @@
+import-module au
+
+$download_page_url = 'http://www.tightvnc.com/download.php'
+
+function global:au_SearchReplace {
+   @{
+        ".\tools\chocolateyInstall.ps1" = @{
+            "(^[$]url32\s*=\s*)('.*')"      = "`$1'$($Latest.URL32)'"
+            "(^[$]url64\s*=\s*)('.*')"      = "`$1'$($Latest.URL64)'"
+            "(^[$]checksum32\s*=\s*)('.*')" = "`$1'$($Latest.Checksum32)'"
+            "(^[$]checksum64\s*=\s*)('.*')" = "`$1'$($Latest.Checksum64)'"
+        }
+    }
+}
+
+function global:au_GetLatest {
+    $page = Invoke-WebRequest $download_page_url
+	$html = $page.parsedHTML
+
+    $download_link = $html.body.getElementsByTagName("a") | where {$_.href -like "http://www.tightvnc.com/download/*.msi"}
+
+    $version = $download_link[0].href.Replace("http://www.tightvnc.com/download/","").Split("/")[0]
+
+    $url64 = $download_link | where {$_.href -like "http://www.tightvnc.com/download/$version*64bit.msi"} 
+    $url32 = $download_link | where {$_.href -like "http://www.tightvnc.com/download/$version*32bit.msi"} 
```

Comment:
Since you only uses links, the previous lines should be changed to be compatible with the `-UseBasicParsing` argument of `Invoke-WebRequest`.
See the [bluefish](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/automatic/bluefish/update.ps1) package for an example.